django_app/redbox_app/templates/accessibility-statement.html

```diff
@@ -1,46 +1,39 @@
-  <p><a href="https://mcmw.abilitynet.org.uk/" class="govuk-link">AbilityNet</a> has advice on making your device easier to use if you have a disability.</p>
+    </ul>
+    <p class="govuk-body">We've also made the website text as simple as possible to understand.</p>
+    <p class="govuk-body"><a href="https://mcmw.abilitynet.org.uk/" class="govuk-link">AbilityNet</a> has advice on making your device easier to use if you have a disability.</p>
+  
     <h2 class="govuk-heading-m">Feedback and contact information</h2>
-    If you need information on this website in a different format like accessible PDF, large print, easy read, audio recording or braille:
-      <ul class="govuk-list govuk-list--bullet">
-        <li>email <a href="mailto:redbox-copilot-support@cabinetoffice.gov.uk" class="govuk-link">redbox-copilot-support@cabinetoffice.gov.uk</a></li>
-      </ul>
-    <p>We'll consider your request and get back to you in 14 days.</p>
-
+    <p class="govuk-body">If you need information on this website in a different format like accessible PDF, large print, easy read, audio recording or braille, email <a href="mailto:redbox-copilot@cabinetoffice.gov.uk" class="govuk-link">redbox-copilot@cabinetoffice.gov.uk</a></p>
+    <p class="govuk-body">We'll consider your request and get back to you in 14 days.</p>
 
     <h2 class="govuk-heading-m">Reporting accessibility problems with this website</h2>
-    <p>We're always looking to improve the accessibility of this website. If you find any problems not listed on this page or think we're not meeting accessibility requirements, contact the team at <a href = "mailto:obt-platform-support@cabinetoffice.gov.uk" class="govuk-link">obt-platform-support@cabinetoffice.gov.uk</a>.</p>
+    <p class="govuk-body">We're always looking to improve the accessibility of this website. If you find any problems not listed on this page or think we're not meeting accessibility requirements, contact the team at <a href="mailto:redbox-copilot@cabinetoffice.gov.uk" class="govuk-link">redbox-copilot@cabinetoffice.gov.uk</a>.</p>
+    
     <h2 class="govuk-heading-m">Enforcement procedure</h2>
-    <p>The Equality and Human Rights Commission (EHRC) is responsible for enforcing the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018 (the "accessibility regulations"). If you're not happy with how we respond to your complaint, <a href="https://www.equalityadvisoryservice.com/" class="govuk-link">contact the Equality Advisory and Support Service (EASS)</a>.</p>
+    <p class="govuk-body">The Equality and Human Rights Commission (EHRC) is responsible for enforcing the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018 (the "accessibility regulations"). If you're not happy with how we respond to your complaint, <a href="https://www.equalityadvisoryservice.com/" class="govuk-link">contact the Equality Advisory and Support Service (EASS)</a>.</p>
```

Comment:
Has the UK withdrawn from EHRC?

Reply:
No idea! @Samuel-Hoskin tagging you in incase you know?

Reply:
The Equality and Human Rights Commission is non-departmental body of the UK Government. It is not tired to EU law

Reply:
We have not withdrawn

Reply:
Thank you @Samuel-Hoskin and @lmwilkigov